### PR TITLE
Increase minSdkVersion

### PR DESCRIPTION
**Pull request by VolodymyrTymets**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-splash-screen",
-  "version": "3.2.0",
+  "version": "3.2.1",
```
Comment (a-tokyo) [inline package.json]:
```suggestion
  "version": "3.2.01",
```

Comment (a-tokyo) [reply package.json]: just like we do here https://github.com/pluralcom/react-native-gifted-chat/blob/main/package.json

**Review by a-tokyo — CHANGES_REQUESTED** [review]
You need to add the auto release gh actions code. Unless you will use the package from github directly, and later migrate to the other splash screen package

File: package.json (hunk 1 above)
Comment (A-Tokyo) [reply package.json]: Unless you will use the package from github directly, and later migrate to the other splash screen package

**Pull request merged**